contributors/guide/scalability-good-practices.md

```diff
@@ -102,68 +96,62 @@ func (c *ControllerX) Run() {
+for resource usage you can either use monitoring tools (heapster + graphana, but at the time we wrote this it stops working at around 100 Nodes), or just plain 'top' on master (which scales as much as you want),
+more data is available on the `/metrics` endpoint of all our components (e.g. the one for the API server contains API call latencies),
+to profile a component create an ssh tunnel to the machine running it, and run `go tool pprof localhost:<your_tunnel_port>` locally
 
 ## Summary
-Summing it up, when writing code you should:
+To summarize, when writing code you should:
 - understand how go manages memory and use it wisely,
-- not LIST from API server,
-- run performance tests in case of huge system-wide changes (e.g. updating big dependencies),
+- not LIST from the API server,
+- run performance tests when making large systemwide changes (e.g. updating big dependencies),
 
-When designing new features/thinking about refactoring you should:
-- when adding new API calls estimate number of additional QPS you'll be sending to the API server,
-- when adding complex logic make sure that you don't add it on a critical path of any basic workflow
+When designing new features or thinking about refactoring you should:
+- when adding new API calls estimate the number of additional QPS you will be sending to the API server,
+- when adding complex logic make sure you do not add it on a critical path of any basic workflow
 
 ## Closing remarks
-We know that thinking about performance impact of changes that all of us write is hard, this is exactly why we want you to help us cater for it, by always having problems that we mentioned here in the back of your mind. In return we'll answer all your question and doubts about possible impact of your changes if you post them either to #sig-scalability Slack channel, or cc @kubernetes/sig-scalability-pr-reviews in your PR/proposal.
+We know that thinking about the performance impact of changes is hard. This is exactly why we want you to help us cater for it, by keeping all the knowledge we have given you here in the back of your mind as you write your code. In return, we'll answer all your question and doubts about possible impact of your changes if you post them either to #sig-scalability Slack channel, or cc @kubernetes/sig-scalability-pr-reviews in your PR/proposal.
 
 * * *
 
-<a name="1">1</a>: If you are using List in tight loops, it's common to be doing that on a subset of a list (field, label, or namespace). Most informers have indices on namespaces, but you may end up needing another index (but only if profile shows it)
+<a name="1">1</a>: If you are using List in tight loops, it is common to do it on a subset of a list (field, label, or namespace). Most informers have indices on namespaces, but you may end up needing another index if profile shows the need.
```

Comment:
LIST ?
"it is common to do so"
"Informers"

Reply:
This is a footnote which refers to the LIST call on Informers, if I am not mistaken. Other changes done.